src/DaPigGuy/PiggyCustomEnchants/enchants/weapons/VampireEnchant.php

```diff
@@ -22,14 +22,6 @@ class VampireEnchant extends ReactiveEnchantment
     /** @var int */
     public $maxLevel = 1;
 
-    /**
-     * @return array
-     */
-    public function getReagent(): array
-    {
-        return [EntityDamageByEntityEvent::class];
-    }
-
```

Comment:
This class should ``use ReactiveTrait`` instead of extending ReactiveEnchantment
> Couldn't comment line 18, so I commented this line